app/components/primer/alpha/tool_tip.ts

```diff
@@ -127,65 +108,11 @@ class ToolTipElement extends HTMLElement {
         animation-timing-function: ease-in;
       }
 
-      :host(.\\:popover-open),
-      :host(.\\:popover-open):before {
+      :host(.\\:popover-open) {
         animation-name: tooltip-appear;
         animation-duration: .1s;
         animation-fill-mode: forwards;
         animation-timing-function: ease-in;
-        animation-delay: .4s;
-      }
-
-      :host(.tooltip-s):before,
-      :host(.tooltip-n):before {
-        right: 50%;
-        margin-right: -${TOOLTIP_ARROW_EDGE_OFFSET}px;
-      }
-      :host(.tooltip-s):before,
-      :host(.tooltip-se):before,
-      :host(.tooltip-sw):before {
-        bottom: 100%;
-        border-bottom-color: var(--bgColor-emphasis, var(--color-neutral-emphasis-plus));
-      }
-      :host(.tooltip-s):after,
-      :host(.tooltip-se):after,
-      :host(.tooltip-sw):after {
-        bottom: 100%
-      }
```

Comment:
I think these ::after classes are still needed to position the empty content that I mentioned [here](https://github.com/primer/view_components/pull/2603/files#r1487030159) about hovering the tooltip text